requirements.txt

```diff
@@ -41,6 +41,7 @@ llama-index-embeddings-huggingface
 llama-index-llms-openai
 llama-index-llms-huggingface
 llama-index-llms-openai-like
+llama_index.llms.ollama
```

Comment:
Thanks @hongsw 
But, I think it should be `llama-index-llms-ollama`.
[pypi link](https://pypi.org/project/llama-index-llms-ollama/)

Reply:
I changed this